--- origsrc/edje-1.7.1/src/bin/Makefile.am	2012-11-10 23:42:07.832668900 -0600
+++ src/edje-1.7.1/src/bin/Makefile.am	2012-11-10 23:41:52.252647400 -0600
@@ -4,7 +4,9 @@ if BUILD_EPP
   SUBDIRS = epp
 endif
 
-bin_SCRIPTS = @EDJE_RECC_PRG@
+if BUILD_EDJE_RECC
+bin_SCRIPTS = edje_recc
+endif
 
 bin_PROGRAMS = @EDJE_CC_PRG@ @EDJE_DECC_PRG@ @EDJE_PLAYER_PRG@ @EDJE_INSPECTOR_PRG@ @EDJE_EXTERNAL_INSPECTOR_PRG@
 
